src/Microsoft.DotNet.Build.Tasks/GenerateAssemblyList.cs

```diff
@@ -31,12 +30,6 @@ public override bool Execute()
                 while ((line = listReader.ReadLine()) != null)
                 {
                     var fileName = Path.GetFileName(line);
-                    if (!processedFileNames.Add(fileName))
```

Comment:
I'm OK with this for now but I don't know the full implications of removing this, we might end up some weird errors in some other places. Something to look out for.